fast64_internal/oot/oot_level.py

```diff
@@ -166,7 +167,16 @@ def onUpdateOOTEmptyType(self, context):
 			setLightPropertyValues(timeOfDayLights.dusk, [120, 90, 0], [250, 135, 50], [30, 30, 60], [120, 70, 50], 0x3E3)
 			setLightPropertyValues(timeOfDayLights.night, [40, 70, 100], [20, 20, 35], [50, 50, 100], [0, 0, 30], 0x3E0)
 
+class OOT_ObjectProperties(bpy.types.PropertyGroup):
+	version: bpy.props.IntProperty(name="OOT_ObjectProperties Version", default=0)
+	cur_version = 0 # version after property migration
+
+	scene: bpy.props.PointerProperty(type=OOTSceneProperties)
+
 oot_obj_classes = (
+	OOTSceneProperties,
+	OOT_ObjectProperties,
+
```

Comment:
extra new line

Reply:
that's on purpose to separate those property groups from whatever is below, it looks somewhat organized somehow

Reply:
fair enough, how come it's OOT_ though but then only OOT for the new one? Might be better to make it consistent

Reply:
There is `Fast64_Properties`, `SM64_Properties`, `OOT_Properties`, `Fast64_ObjectProperties`, `SM64_ObjectProperties`, ...
So for consistency with the SM64 counterpart at least (`SM64_ObjectProperties`) I chose `OOT_ObjectProperties`

Ideally `OOTSceneProperties` would be `OOT_SceneProperties` but then per this whole naming scheme one may mistakenly think "Scene" is `bpy.types.Scene`, not a OoT scene
Simply removing the underscore doesn't entirely address it :/

Reply:
classes can be renamed without compatibility issues for blend files so let's just go with this and rename later if someone has a better idea